outreach_announcement.md

```diff
@@ -1,15 +1,29 @@
+Most UofT Coders sessions are hour-long tutorials where one person from our community leads a hands-on demonstration of a tool, package or piece of code, while everyone else follows along. We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set. 
 
-Our next meeting will be <event title> on September 10, 2015 at 6 pm in MP408 (McLennan Physical Laboratories). Everyone is welcome, regardless of experience level. If you can't make it out but have ideas or questions, please post a thread on our [issue tracker](https://github.com/UofTCoders/studyGroup/issues) or check out the group [website](
-http://uoftcoders.github.io/studyGroup/). Also, if you could take a second to fill out [this survey](https://docs.google.com/forms/d/1QIRsnaJjpf5sTV_nMC9KEA44Mz21wuASDxttdgTBj44/viewform?usp=send_form) to let us know what you'd be interested in, that would be a big help, too. Finally, if you'd like updates on what our Study Group is doing, watch our [events repository](https://github.com/UofTCoders/Events) by clicking 'Watch' at the top of our Github repo. 
+Our next meeting will be **event title** on **date** at 6 pm in MP408 (McLennan Physical Laboratories). If you can't make it out but have ideas or questions, please post a thread on our [issue tracker](https://github.com/UofTCoders/studyGroup/issues) or check out the group [website](
+https://uoftcoders.github.io/). Finally, if you'd like to stay informed about upcoming events, you can follow our [Google Calendar](https://calendar.google.com/calendar/embed?src=uoftcoders@gmail.com), sign up for our [email list](https://groups.google.com/forum/#!forum/uoftcoders), or watch our [events repository](https://github.com/UofTCoders/Events) by clicking 'Watch' at the top of our Github repo. 
 
 Thanks!
 
-<Name>
-<Department>
+**Name** on behalf of UofT Coders
+
 University of Toronto
-<email>
\ No newline at end of file
+
+uoftcoders@gmail.com
+
+
+------------------
+
+## Shorter blurb (i.e. GSU newsletter)
+
+UofT Coders: Wednesdays at 6 pm
+
+UofT Coders is a group for researchers and learners to get together, share coding skills, and help each other out with the programming many have to do as part of research. Topics from across the spectrum of computing and research practices are covered. UofT Coders meets on Wednesdays at 6:00 pm in MP408. To receive updates on future sessions, follow the UofTCoders Events repository on Github (https://github.com/UofTCoders/Events) or send an email to uoftcoders@gmail.com. 
```

Comment:
"programming many have to do as part of research." -> "programming we do as part of our research."